2.3.4/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+Astronomer Certified 2.3.4-4, TBD
+---------------------------------
+
+### Bugfixes
+
+- Fix backporting issue with faulty executor config serialization logic
+
```

Comment:
Any idea what to cherrypick to this release would be appreciated. For now, it has only this fix on backport

Reply:
Let's just go that for now.